0003-minimal-snarkyjs-checks-part1.md

```diff
@@ -0,0 +1,91 @@
+## Summary
+[summary]: #summary
+
+A minimal layer of SnarkyJS checks that won't break with typical protocol development.
```

Comment:
Sentence fragment

How about:

"We describe a minimal layer of SnarkyJS check that won't hinder typical protocol development."